mito-ai/style/ChatTaskpane.css

```diff
@@ -112,7 +112,7 @@
 
 .chat-input {
     outline: none;
-    border: none;
+    border: 1px solid var(--purple-500);
```

Comment:
Maybe I'm wrong, but this input always felt like it blended into the gray background.

Reply:
Nice, yeah I think a border looks good here. When I add more colors to the taskpane in my next few PR's I'll play around with which ones look best. 

I like that we're using these color variables!